README.md

```diff
@@ -50,6 +50,10 @@ git push origin refs/tags/vX.Y.Z
 
 Wait for CI to build and publish the release (Github Actions).
 
+### Building docs
+
+If you have `mise` installed, you should have all the tooling you need for the docs engine.  Just run `bun run docs:dev` from the `docs` directory to enter the watch-rebuild cycle to test your docs changes.
```

Comment:
I could've sworn I made a mise task for this but if I didn't we should

Reply:
added -- you did not